tests/btp/k8s-resources/simple-app/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Do you need this file? It looks like it's unused

Reply:
Indeed, I was sure that it is used because I got an error sth like ".. no kustomization file.." but I could not recreate that, and it is working without it. Thanks for that